grpclb/src/main/java/io/grpc/grpclb/GrpclbLoadBalancerProvider.java

```diff
@@ -78,31 +80,42 @@ public ConfigOrError parseLoadBalancingPolicyConfig(
   ConfigOrError parseLoadBalancingConfigPolicyInternal(
       Map<String, ?> rawLoadBalancingPolicyConfig) {
     if (rawLoadBalancingPolicyConfig == null) {
-      return ConfigOrError.fromConfig(DEFAULT_MODE);
+      return ConfigOrError.fromConfig(GrpclbConfig.create(DEFAULT_MODE));
     }
+    Object rawServiceName = rawLoadBalancingPolicyConfig.get("serviceName");
```

Comment:
I'd have expected this here:
```java
String serviceName = JsonUtil.getString(rawLoadBalancingPolicyConfig, "serviceName");
```

It produces a better error message when the value is not the correct type.

But it also looks like this code was avoiding the JsonUtil (e.g., the `getList()` just below). I think that may have been because it was ServiceConfigUtil before...? But it uses `ServiceConfigUtil.unwrapLoadBalancingConfigList`...

Reply:
done. i am not sure why it was not using `JsonUtil`.